internal/runners/export/config.go

```diff
@@ -0,0 +1,96 @@
+package export
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+)
+
+// Filter is the --filter flag for the export config command, it implements captain.FlagMarshaler
+type Filter int
+
+const (
+	Unknown Filter = iota
+	Dir
+)
+
+var lookup = map[Filter]string{
```

Comment:
```suggestion
var filterLookup = map[Filter]string{
```

This packages isn't only about filters.